cmd/aws-efs-csi-driver-operator/main.go

```diff
@@ -4,10 +4,11 @@ import (
 	"context"
 	"os"
 
-	"github.com/openshift/csi-operator/pkg/driver/aws-efs"
+	aws_efs "github.com/openshift/csi-operator/pkg/driver/aws-efs"
```

Comment:
Looks as NOOP change (make succeeds with or without this change)

Reply:
fair point. But lets fix this in a follow up, I do not want to run entire battery of tests for this change.